ui/docs/manual/deploying/anonymous-role.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Anonymous Role
+---
+
+# Anonymous Role
+
+A single scope `assume:anonymous` will be granted for all calls (with or without credentials).
+The `anonymous` role is aimed at private Taskcluster installations. With this, it will
+be possible to prevent any useful access to the Taskcluster deployment without credentials
+allowing for greater administrator control over "public" endpoints.
```

Comment:
Maybe add a `## Private Deployments` here, for symmetry, suggesting leaving the role empty.